api/controllers/MobileAppController.js

```diff
@@ -12,26 +26,56 @@ module.exports = {
       iTunesItemIdentifier: '1002185140'
     }
     */
+    var SUGGEST = 'suggest'
+    var FORCE = 'force'
+    var IOS = 'ios'
+    var ANDROID = 'android'
+    var result = {}
 
-    switch (req.headers['ios-version']) {
-      case '1.6':
-      case '1.65':
-        result = {
-          type: 'force',  // can be 'force' or 'suggest'
-          title: 'A new version of the app is available',
-          message: 'The version you are using is no longer compatible with the site. Please go to the App Store now to update',
-          iTunesItemIdentifier: '1002185140'
-        }
+    switch (req.param('ios-version')) {
+      case '2.0':
+        result = resultBuilder(SUGGEST, IOS)
```

Comment:
you want to let the 2.0 versions pass through without interruption, no? and basically force everything else?